### ISSUE TITLE
[REQUEST] How do we enable deepLinking?

### ISSUE THREAD

**Issue by roxanemace** [rswag/rswag#588]
## Is your feature request related to a problem? Please describe.
Hello, I would like to know how to enable [`deepLinking` on the Swagger UI](https://github.com/swagger-api/swagger-ui/blob/cc408812fc927e265da158bf68239530740ab4cc/docs/usage/deep-linking.md#deeplinking-parameter). Do I must [custom the Swagger UI ](https://github.com/rswag/rswag#customizing-the-swagger-ui) ?

## Describe the solution you'd like
It's more a question than I request.

## What support could we give you, so you could implement this yourself?
Not applicable.

## Describe alternatives you've considered
Not applicable.

## Additional context
Not applicable.

## Relates to which version of OAS (Open API Specification)
- [ ] OAS2
- [X] OAS3
- [ ] OAS3.1

**Comment by v-v-b** [comment]
ran into the same issue recently,
ended up with this (dirty) workaround:

```ruby
Rswag::Ui.configure do |c|
  # other stuff...
  c.config_object.merge!({ "deepLinking" => true })
end
```

**Comment by pboling** [comment]
Bad bot.

**Comment by pboling** [comment]
Stale bot is going to kill this project...

**Comment by jtannas** [comment]
> Stale bot is going to kill this project...

Yeah... We added it after getting maintainer permissions due to all of the issues that were years old and no longer relevant. The intent was that we'd then keep up on the newer stuff.
Things haven't really turned out that way :-/

@romanblanco Think we should turn it off?

**Comment by romanblanco** [comment]
> @romanblanco Think we should turn it off?

@jtannas, I agree with turning it off, https://github.com/probot/stale is even deprecated now.
